tasks/e2e-kitchensink.sh

```diff
@@ -21,8 +21,10 @@ temp_app_path=`mktemp -d 2>/dev/null || mktemp -d -t 'temp_app_path'`
 
 function cleanup {
   echo 'Cleaning up.'
+  ps -ef | grep 'react-scripts' | grep -v grep | awk '{print $2}' | xargs kill -s 9
```

Comment:
There are left over node processes after running kitchensink (even on macOS), so this tries to clean them up.

It also aims to fix a `Device or resource busy` message from windows when trying to delete the directory, which it does not... I'm not sure why. 😢